services/api.py

```diff
@@ -973,8 +984,16 @@ def list(self, request, *args, **kwargs):
             else:
                 setattr(self, key, None)
 
-        input_val = request.query_params.get('input', '').strip()
-        q_val = request.query_params.get('q', '').strip()
+        # query parameter validation
+        f = {k: v[0].split(',') for k, v in filters.lists()}
```

Comment:
Especially here in the text search, where the input comes directly from the user, we cannot split all fields by commas -- the user can type a comma as part of `q` or `input` and it shouldn't be a problem...